.github/workflows/bbw_build_container.yml

```diff
@@ -105,6 +105,11 @@ jobs:
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le
             # //TEMP chicken/egg problem - could add s390x
             # but that would need a galera-4 build from on this image
+          - dockerfile: opensuse.Dockerfile pip.Dockerfile
```

Comment:
I will integrate both sles/opensuse also in my workflow refactoring:
https://github.com/MariaDB/buildbot/compare/dev...RazvanLiviuVarzaru:buildbot-r:dev

But I prefer merging this pull request first because the refactored workflow will not push to quay, not until we validate it as a successful project.